apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -451,8 +450,8 @@ internal TestSubscriptionHandler(bool ackOrNack)
                 Responses = new List<AckNackResponse>();
             }
 
-            public override async Task<SubscriberClient.Reply> HandleMessage(PubsubMessage message, CancellationToken cancellationToken) =>
-                await Task.FromResult(_ackOrNack ? SubscriberClient.Reply.Ack : SubscriberClient.Reply.Nack);
+            public override Task<SubscriberClient.Reply> HandleMessage(PubsubMessage message, CancellationToken cancellationToken) =>
+                Task.FromResult(_ackOrNack ? SubscriberClient.Reply.Ack : SubscriberClient.Reply.Nack);
```

Comment:
tiny nit: but `_ackOrNack` => `_ack`?

Reply:
Discussed offline - we're going to leave this for now.